exercises/sum-of-multiples/canonical-data.json

```diff
@@ -17,6 +17,13 @@
       "expected": 3
     },
     {
+      "description": "multiples of 3 up to 7",
```

Comment:
Great 👍 

Is this the best place for this test? 
Is it simpler than the earlier cases?
Only having 1 factor makes it look simpler to me, but I'm not sure.

Reply:
I personally think it looks a little weird going back and forth between multiples of 3 and 5, and then a single 3 -- but following the approach I gave earlier, it makes the most sense. Again, to solve Test 1 and Test 2 all you're doing is:

```
    public static int Sum(IEnumerable<int> multiples, int max)
    {
        if(max > 3)
        {
            return 3;
        }

        return 0;
    }
```

That's a big reason to adjust the previous tests, mostly because I think you just completely ignore the 5 case anyway.

Reply:
@Insti What do you feel about @jpreese's explanation? Is that okay? If so, is this ready to be merged or are there still changes to me made?

Reply:
Yes, this is fine.
The PR as it is currently is fine to be merged.